### PR TITLE
feat: finalise heroku support

### PR DESCRIPTION
**Pull request by nullishamy**
This will contain all of the changes required to make the project run on Heroku.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "NODE_ENV=development ts-node-dev src/index.ts | pino-pretty",
     "build": "prisma generate && tsc",
+    "heroku-postbuild": "prisma migrate deploy && prisma generate && tsc",
```
Comment (BobobUnicorn) [inline package.json:9]: note: figure out why `prisma migrate deploy` doesn't create the tables.

Comment (nullishamy) [reply package.json:9]: Should we just use `prisma db push` until we find a better way?

Comment (BobobUnicorn) [reply package.json:9]: figured it out

Comment (BobobUnicorn) [reply package.json:9]: we need to run `prisma migrate dev` to generate the migration scripts

**Review by BobobUnicorn — APPROVED** [review]

**Pull request merged**